docs/io.cozy.files.md

```diff
@@ -53,7 +53,8 @@ The attributes of a file are:
 
 It also has a relationship with its `parent` in the JSON-API representation.
 
-For an `image`, there are 3 links to thumbnails: `small`, `medium`, and `large`.
+For an `image`, there are 3 links to thumbnails: `small`, `medium`, and `large`. 
+Theses thumbnails' links are valid for 10 minutes. 
```

Comment:
```suggestion
These thumbnails' links are valid for 10 minutes. 
```

Reply:
After 10 minutes, what happens ? The links send back 404 ou 500 ?

Reply:
The secret contained in the link is not valid anymore, I assume I will have a "wrong token" https://github.com/cozy/cozy-stack/blob/master/web/files/files.go#L815, so a 400.

Reply:
Thanks for the addition 👍